api/v1/types.go

```diff
@@ -80,6 +80,9 @@ const (
 	// PowerStore - placeholder for constant powerstore
 	PowerStore DriverType = "powerstore"
 
+	// PowerStoreName - placeholder for constant PowerStore
+	PowerStoreName DriverType = "powerstore"
+
```

Comment:
@kk3, why are we adding PowerStoreName when we have PowerStore at Line No 81. Also the constant name is same in both the cases